subscription-manager.spec

```diff
@@ -4754,12 +4760,6 @@ gtk-update-icon-cache -f %{_datadir}/icons/hicolor &>/dev/null || :
 
 * Mon Dec 05 2011 William Poteat <wpoteat@redhat.com> 0.98.5-1
 - 755031: Unregister before attempting to run a second registration
-  (jbowes@redhat.com)
-
-* Mon Dec 05 2011 William Poteat <wpoteat@redhat.com> 0.98.4-1
```

Comment:
We will get a :frowning_face: if we change the changelog.  Please back this section out,